frontend/src/Explorer/View/Dashboard/Blocks.purs

```diff
@@ -53,3 +59,40 @@ emptyBlocksView message =
     P.div
         [ P.className CSS.blocksWaiting ]
         [ P.text message ]
+
+currentBlocks :: State -> CBlockEntries
+currentBlocks state =
+    if expanded
+    then slice minBlockIndex (minBlockIndex + maxBlockRows) blocks
+    else slice 0 minBlockRows blocks
+    where
+        blocks = unwrapLatestBlocks $ state ^. latestBlocks
+        expanded = state ^. dashboardBlocksExpanded
+        currentBlockPage = state ^. (dashboardViewState <<< dbViewBlockPagination)
+        minBlockIndex = (currentBlockPage - 1) * maxBlockRows
+
+blocksFooterView :: State -> P.Html Action
+blocksFooterView state =
+    if expanded then
+        paginationView { label: translate (I18nL.common <<< I18nL.cOf) $ lang'
+                        , currentPage: currentBlockPage
+                        , maxPage: (+) minPagination <<< flip (/) maxBlockRows $ length blocks
```

Comment:
@ksaric The `paginationView` is already a component, which has no dependencies to state, it takes just properties https://github.com/input-output-hk/cardano-sl-explorer/blob/fix/CSE-70-search-invalid-state/frontend/src/Explorer/View/Common.purs#L201-L266 

The same for table header (`txHeaderView`) and table body (`txBodyView`):
- `txBodyView` https://github.com/input-output-hk/cardano-sl-explorer/blob/fix/CSE-70-search-invalid-state/frontend/src/Explorer/View/Common.purs#L141-L154
- `txHeaderView` https://github.com/input-output-hk/cardano-sl-explorer/blob/fix/CSE-70-search-invalid-state/frontend/src/Explorer/View/Common.purs#L83-L104